independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/AnnotationsTransformationContext.java

```diff
@@ -57,12 +67,29 @@ void setAnnotations(C annotations) {
     }
 
     public Collection<AnnotationInstance> getAllAnnotations() {
+        return getAllAnnotationForTarget(getTarget());
+    }
+
+    public Collection<AnnotationInstance> getAllAnnotationTargetAnnotations() {
```

Comment:
The name of this method is a mouthful. How about removing one "annotation" and using just `getAllTargetAnnotations()`?

Reply:
Oh yeah, good point. I dunno :shrug: